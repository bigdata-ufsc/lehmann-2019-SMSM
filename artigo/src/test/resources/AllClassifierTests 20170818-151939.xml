<?xml version="1.0" encoding="UTF-8"?><testrun name="AllClassifierTests" project="artigo" tests="4" started="4" failures="0" errors="0" ignored="0">
  <testsuite name="br.ufsc.lehmann.classifier.AllClassifierTests" time="89167.58">
    <testsuite name="br.ufsc.lehmann.classifier.MSMMoveEllipsesClassifierTest" time="54309.87">
      <testsuite name="[Dublin bus(lines=2)[STOP_STREET_NAME]]" time="54309.87">
        <testcase name="validation_accuracy[Dublin bus(lines=2)[STOP_STREET_NAME]]" classname="br.ufsc.lehmann.classifier.MSMMoveEllipsesClassifierTest" time="26614.862"/>
        <testcase name="validation_precision_recall[Dublin bus(lines=2)[STOP_STREET_NAME]]" classname="br.ufsc.lehmann.classifier.MSMMoveEllipsesClassifierTest" time="27695.008"/>
      </testsuite>
    </testsuite>
    <testsuite name="br.ufsc.lehmann.classifier.MSMMovePointsClassifierTest" time="34857.71">
      <testsuite name="[Dublin bus(lines=2)[STOP_STREET_NAME]]" time="34857.71">
        <testcase name="validation_accuracy[Dublin bus(lines=2)[STOP_STREET_NAME]]" classname="br.ufsc.lehmann.classifier.MSMMovePointsClassifierTest" time="17628.772"/>
        <testcase name="validation_precision_recall[Dublin bus(lines=2)[STOP_STREET_NAME]]" classname="br.ufsc.lehmann.classifier.MSMMovePointsClassifierTest" time="17228.938"/>
      </testsuite>
    </testsuite>
  </testsuite>
</testrun>
