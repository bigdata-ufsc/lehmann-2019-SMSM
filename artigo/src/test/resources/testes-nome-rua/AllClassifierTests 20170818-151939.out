Executing SQL...
Fetching...
Loaded 143382 GPS points from database
Loaded 237 trajectories from database
Semantic Trajectories statistics: mean - 604,99, min - 1,00, max - 4226,00, sd - 978,48
MSMMoveEllipsesClassifierTest#validation_accuracy[Dublin bus(lines=2)[STOP_STREET_NAME]]
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
1.0
0.9914893617021278
MSMMoveEllipsesClassifierTest#validation_precision_recall[Dublin bus(lines=2)[STOP_STREET_NAME]]
[1.0, 1.0, 1.0, 1.0, 0.0, 0.0]
[0.99, 1.0, 0.9948717948717949, 0.9555555555555555, 0.04444444444444444, 0.01]
MSMMovePointsClassifierTest#validation_accuracy[Dublin bus(lines=2)[STOP_STREET_NAME]]
1.0
1.0
MSMMovePointsClassifierTest#validation_precision_recall[Dublin bus(lines=2)[STOP_STREET_NAME]]
[1.0, 1.0, 1.0, 1.0, 0.0, 0.0]
[1.0, 0.9816067653276956, 0.9906403940886699, 1.0, 0.0, 0.0]
